chainio/clients/elcontracts/writer.go

```diff
@@ -290,3 +302,30 @@ func (w *ELChainWriter) DepositERC20IntoStrategy(
 	w.logger.Infof("deposited %s into strategy %s", amount.String(), strategyAddr)
 	return receipt, nil
 }
+
+func (w *ELChainWriter) SetClaimerFor(
+	ctx context.Context,
+	claimer gethcommon.Address,
+) (*gethtypes.Receipt, error) {
+	if w.rewardsCoordinator == nil {
+		return nil, errors.New("RewardsCoordinator contract not provided")
+	}
+
+	w.logger.Infof("setting claimer %s", claimer)
```

Comment:
let's remove all logs in sdk. We don't need this. we give full error context when errors happen anyways.

Reply:
599a3176240a861d0d3986348b300a784e596666